runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkRunnerResult.java

```diff
@@ -27,19 +27,18 @@
 
 /**
  * Result of executing a {@link org.apache.beam.sdk.Pipeline} with Flink. This
- * has methods to query to job runtime and the final values of
- * {@link org.apache.beam.sdk.transforms.Aggregator}s.
+ * has methods to query to job runtime and the final values of the metrics.
  */
 public class FlinkRunnerResult implements PipelineResult {
 
-  private final Map<String, Object> aggregators;
+  private final Map<String, Object> metrics;
```

Comment:
This also holds for all the places below, where you replaced `aggregators` or `aggregator` by `metrics` or `metric`.